src/app/home/home.component.html

```diff
@@ -13,12 +13,15 @@
             <ag-grid-angular
               style="width: 1100px; height: 1700px; padding: 1px;"
               class="ag-theme-alpine"
-              [rowData]="StateData['statewise']"
+              [rowData]="this.DistrictData"
```

Comment:
you don't need to use 'this' in the HTML file. It always takes the current instance of that variable